yktrace/src/tir.rs

```diff
@@ -285,74 +258,143 @@ impl TirTrace {
             }
         }
 
+        // Remove the remnants of the `start_tracing` (1 instruction) and `stop_tracing` (5
+        // instructions) calls at the beginning and end of the trace.  To be sure we don't remove
+        // something important, let's put some asserts in to check these instructions are always
+        // roughly the same.
+        match ops.pop() {
+            Some(TirOp::Statement(Statement::Enter(CallOperand::Fn(s),_,_,_))) => {
+                assert!(s.contains("stop_tracing"))
```

Comment:
Did you think about if these should be ,`debug_assert!`?

Reply:
These can probably be `debug_assert"s, yes.

Reply:
Fixed in 51ea6c0